calcom/cal.com
# fix: meeting ended and started webhooks trigger

## What does this PR do?

This fixes several issues with the Meeting ended and Meeting started trigger. The original issue was that if one booking was canceled, all scheduled triggers for that webhook, also the ones that were not related to the canceled booking, were deleted. This is why the meeting ended/started trigger didn't always trigger. 

Cleans up some old code. Since we use `WebhookScheduledTriggers` with a corn job, `scheduledJobs` are not needed anymore. 

Fixes #14685

## How should this be tested?

- Make sure you have existing bookings
- Create webhook with meeting ended and meeting started trigger
- See that WebhookScheduledTriggers are created for all bookings for each trigger 
- Cancel one of the bookings 
- See that only the WebhookScheduledTriggers related to the booking were related

You can also test: 
- delete webhook 
- activate/deactivate webhook
- edit webhook triggers
